### PR TITLE
Additional parsers

### PR DESCRIPTION
**Pull request by amandadumi**
This incorporates the skip_line functionality from logfileparser   as a class function into file_handler.  This function might only ever be used in a virtual way, so the extra argument I've added might not be needed, but I wanted to be safe.  

This is adding atomnos and atomcoords.

### CODE REVIEW

**Comment by amandadumi** [comment]
Questions:
1. Should i be formatting with black now?
2. Can we discuss what is the plan for porting tests? (or if this was discussed and I'm forgetting, please remind me.)

**Comment by shivupa** [comment]
> Questions:
> 
>     1. Should i be formatting with black now?

Let's do this once on `main` at the end.

>     2. Can we discuss what is the plan for porting tests? (or if this was discussed and I'm forgetting, please remind me.)

This needs to be a topic of discussion on tuesday

File: cclib/combinator/combinator.py [hunk 1]
```diff
@@ -1,5 +1,5 @@
 from dataclasses import dataclass
-from cclib.parser_properties import scfenergies
+import cclib.parser_properties as cpp
```
Comment (shivupa) [inline cclib/combinator/combinator.py:2]: lol cpp is very close to `c++`

Comment (shivupa) [reply cclib/combinator/combinator.py:2]: nothing to change

File: cclib/parser_properties/atomcoords.py [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+        return None
+
+    @staticmethod
+    def parse(file_handler, program:str, ccdata) -> np.ndarray:
+        constructed_data = None
+        if program in atomcoords.known_codes:
+            file_handler.virtual_set()
+            program_parser = getattr(atomcoords, program)
+            constructed_data = program_parser(file_handler, ccdata)
+            file_handler.virtual_reset()
```
Comment (shivupa) [inline cclib/parser_properties/atomcoords.py:35]: it feels like this might be a repeated paradigm can we factor this out? nothing to change right now

**Review by shivupa — APPROVED** [review]

File: cclib/combinator/combinator.py (hunk 1 above)
Comment (berquist) [inline cclib/combinator/combinator.py:2]: "This triggers me for obvious reasons"
/click resolved conversation/
"Apparently I'm not the only one"

**Review by berquist — APPROVED** [review]

**Comment by berquist** [comment]
Formatting will be done after the rebase.

Tests I'm still not sure how this will work with the new pytest stuff which I increasingly think should be merged into `master` and not `main`.

**Pull request merged**